.github/workflows/tests.yaml

```diff
@@ -0,0 +1,35 @@
+name: Test analysis notebooks
+
+on: [push, pull_request]
+
+jobs:
+  run-analysis:
+    name: Test analysis notebooks
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+
+      - name: Setup conda environment
+        run: |
+          # $CONDA is an environment variable pointing to the root of the miniconda directory
+          $CONDA/bin/conda env update --file environment.yml --name base
+          $CONDA/bin/pip install .
+          $CONDA/bin/pip install pytest
+          $CONDA/bin/pip install nbval
+      #    export LD_LIBRARY_PATH=$(python -m rpy2.situation LD_LIBRARY_PATH):${LD_LIBRARY_PATH}
```

Comment:
Is Github Actions failing to find rpy2 or the R installation itself? 

If it's the latter you may be able to manually specify the R_HOME variable with
``` bash 
R_HOME=`R RHOME`
```

Reply:
Or maybe 
``` bash
R_HOME=`$CONDA/bin/R RHOME`
```
because github actions paths are a nightmare

Reply:
The error message I was getting was `ValueError: r_home is None. Try python -m rpy2.situation` which I assumed meant that it wasn't finding R. 

Thank you for your suggestions, I tried them but they sadly didnt' work.